pkg/persistence/config/internal/persistence/type_definition.go

```diff
@@ -379,3 +378,16 @@ func (c TypeDefinition) MarshalYAML() (interface{}, error) {
 	}
 	return nil, fmt.Errorf("unknown type: %T", c.Type)
 }
+
+// getAllUserPermission returns the allUsers permission and falls back to "none" if permission are set but allUsers is missing
+func getAllUserPermission(p *PermissionDefinition) *config.AllUserPermissionKind {
```

Comment:
I really like it with a separate method and an explanation of what is going on here 👍